api/src/main/java/org/openmrs/module/commonreports/reports/DRCARTPDVReportManager.java

```diff
@@ -0,0 +1,268 @@
+
+import org.openmrs.Concept;
+import org.openmrs.VisitType;
+import org.openmrs.api.ConceptService;
+import org.openmrs.api.VisitService;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.commonreports.ActivatedReportManager;
+import org.openmrs.module.commonreports.CommonReportsConstants;
+import org.openmrs.module.initializer.api.InitializerService;
+import org.openmrs.module.reporting.cohort.definition.CodedObsCohortDefinition;
+import org.openmrs.module.reporting.cohort.definition.CompositionCohortDefinition;
+import org.openmrs.module.reporting.cohort.definition.GenderCohortDefinition;
+import org.openmrs.module.reporting.cohort.definition.NumericObsCohortDefinition;
+import org.openmrs.module.reporting.cohort.definition.BirthAndDeathCohortDefinition;
+
+import org.openmrs.module.reporting.cohort.definition.SqlCohortDefinition;
+import org.openmrs.module.reporting.cohort.definition.VisitCohortDefinition;
+import org.openmrs.module.reporting.common.MessageUtil;
+import org.openmrs.module.reporting.common.RangeComparator;
+import org.openmrs.module.reporting.common.SetComparator;
+import org.openmrs.module.reporting.common.DurationUnit;
+
+import org.openmrs.module.reporting.dataset.definition.CohortCrossTabDataSetDefinition;
+import org.openmrs.module.reporting.evaluation.parameter.Mapped;
+import org.openmrs.module.reporting.evaluation.parameter.Parameter;
+import org.openmrs.module.reporting.report.ReportDesign;
+import org.openmrs.module.reporting.report.definition.ReportDefinition;
+import org.openmrs.module.reporting.report.manager.ReportManagerUtil;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Component;
+import org.openmrs.module.reporting.cohort.definition.BaseObsCohortDefinition.TimeModifier;
+import org.openmrs.module.reporting.cohort.definition.AgeCohortDefinition;
+import org.openmrs.module.reporting.common.DateUtil;
+
+@Component
+public class DRCARTPDVReportManager extends ActivatedReportManager {
+	
+	@Autowired
+	private InitializerService inizService;
+	
+	@Override
+	public boolean isActivated() {
+		//return inizService.getBooleanFromKey("report.drc.hivStage.active", false);
+		return true;
+		
+	}
+	
+	@Override
+	public String getVersion() {
+		return "1.0.0-SNAPSHOT";
+	}
+	
+	@Override
+	public String getUuid() {
+		return "7e2e8afb-46cf-48a7-8195-d6ce877b30b8";
+	}
+	
+	@Override
+	public String getName() {
+		return MessageUtil.translate("commonreports.report.drc.artPDV.reportName");
+	}
+	
+	@Override
+	public String getDescription() {
+		return MessageUtil.translate("commonreports.report.drc.artPDV.reportDescription");
+	}
+	
+	private Parameter getReportingDateParameter() {
+		String today = new SimpleDateFormat("yyyy-MM-dd").format(new Date());
+		return new Parameter("onOrBefore", "Reporting End Date", Date.class, null, DateUtil.parseDate(today, "yyyy-MM-dd"));
```

Comment:
Do we need to translate parameter names?

Reply:
Let me do that for this.

Reply:
Fixed